include/swift/SIL/OwnershipUseVisitor.h

```diff
@@ -389,7 +389,7 @@ bool OwnershipUseVisitor<Impl>::visitGuaranteedUse(Operand *use) {
   case OperandOwnership::UnownedInstantaneousUse:
   case OperandOwnership::BitwiseEscape:
   case OperandOwnership::EndBorrow:
-    return handleUsePoint(use, UseLifetimeConstraint::NonLifetimeEnding);
+    return handleUsePoint(use, UseLifetimeConstraint::LifetimeEnding);
```

Comment:
Are those other (besides `::EndBorrow`) `OperandOwnership`s (i.e. `::InstantaneousUse`, `::ForwardingUnowned`, ...) also `LifetimeEnding`?

Reply:
Ah no, thanks for catching this